sklearn/preprocessing/data.py

```diff
@@ -270,8 +293,16 @@ def transform(self, X):
         check_is_fitted(self, 'scale_')
 
         X = check_array(X, copy=self.copy, ensure_2d=False)
-        X *= self.scale_
-        X += self.min_
+
+        if len(X.shape) == 2:
```

Comment:
> The condition isn't met when the passed in array is 1-d.

That's an invalid input. We are not systematically raising errors in
scikit-learn for such an input, but we will be moving toward that. In the
mean time, using check_X_y will make sure that X is 2D.